<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="GIGANEW-PC" ignored="0" name="Test_ff" tests="1" failures="1" timestamp="01 Apr 2018 04:19:37 GMT" time="16.119" errors="0">
  <testcase name="test_verifySuccessfullLogin" time="1.513" classname="scripts.TestScript_02">
    <failure type="java.lang.AssertionError" message="expected [Username or Password is invalid. Please try again..] but found [Username or Password is invalid. Please try again.]">
      <![CDATA[java.lang.AssertionError: expected [Username or Password is invalid. Please try again..] but found [Username or Password is invalid. Please try again.]
at pom.Login.verifyErrorMsg(Login.java:54)
at scripts.TestScript_02.test_verifySuccessfullLogin(TestScript_02.java:24)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test_verifySuccessfullLogin -->
</testsuite> <!-- Test_ff -->
